`mypy` complains like this:
```
mfabian@hathi:/local/mfabian/src/ibus-table (release-candidate-1.17.13 $)
$ ./mypy.sh 
running /local/mfabian/src/ibus-table/mypy.sh
Checking /local/mfabian/src/ibus-table/engine
tabcreatedb.py:352: error: Item "None" of "Match[str] | None" has no attribute "groups"  [union-attr]
Found 1 error in 1 file (checked 11 source files)
Checking /local/mfabian/src/ibus-table/setup
Success: no issues found in 5 source files
Checking /local/mfabian/src/ibus-table/tests
Success: no issues found in 4 source files
mfabian@hathi:/local/mfabian/src/ibus-table (release-candidate-1.17.13 $)
```

So I think I’ll add this to your patch to keep `mypy` happy:

```diff
$ git diff 
diff --git a/engine/tabcreatedb.py b/engine/tabcreatedb.py
index 32b0135..8dcd858 100644
--- a/engine/tabcreatedb.py
+++ b/engine/tabcreatedb.py
@@ -349,7 +349,10 @@ def main() -> None:
 
     def attribute_parser(f: Iterable[str]) -> Iterable[Tuple[str, str]]:
         for line in f:
-            attr, val = re.fullmatch(r'(.*?)==?(.*)', line).groups()
+            match = re.fullmatch(r'(.*?)==?(.*)', line)
+            if match is None:
+                raise ValueError(f"Invalid attribute format: {line!r}")
+            attr, val = match.groups()
             attr = attr.strip().lower()
             val = val.strip()
             yield (attr, val)
```

--- a/engine/tabcreatedb.py
+++ b/engine/tabcreatedb.py
@@ -349,7 +349,10 @@ def main() -> None:
 
     def attribute_parser(f: Iterable[str]) -> Iterable[Tuple[str, str]]:
         for line in f:
-            attr, val = re.fullmatch(r'(.*?)==?(.*)', line).groups()
+            match = re.fullmatch(r'(.*?)==?(.*)', line)
+            if match is None:
+                raise ValueError(f'Invalid attribute format: {line!r}')
+            attr, val = match.groups()
             attr = attr.strip().lower()
             val = val.strip()
             yield (attr, val)

